src/tests/UnitTests/Logging/Mocks/DummyLogService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using CodeMonkeys.Logging;
+
+using System;
+
+namespace CodeMonkeys.UnitTests.Logging.Mocks
+{
+    internal class DummyLogService :
+        ScopedLogService<DummyLogOptions>
+    {
+        public DummyLogService(
+            string context)
+
+            : base (context)
+        {
+
+        }
+
+
+        protected override void PublishMessage(
+            LogMessage message)
+        {
+            throw new NotImplementedException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement this method or throw 'NotSupportedException' instead.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5645023)